src/components/HfHotDogdBannerHomePage/index.vue

```diff
@@ -0,0 +1,27 @@
+<script>
+
+</script>
+<template>
+    <div class="container hotdog-banner">
+        <div class="wide">
+            <img src="@/assets/images/banner-image3.png" alt="" class="hotdog-image">
+            <div class="banner-info">
+                <h1 class="h1-hotfresh">Hot Fresh</h1>
+                <h1 class="h1-hotdog">HOTDOG</h1>
+                <HfButton title="ĐẶT HÀNG NGAY" class="btn-order" />
+            </div>
+            <div class="banner-think">
+                <img src="@/assets/images/banner-image5.png" alt="think">
+                <p>
+                    Chỉ với <br>
+                    <span>50.000VND</span>
+                </p>
+            </div>
+            <div class="banner-thunk">
+                <img src="@/assets/images/banner-image6.png" alt="thunk">
+                <img src="@/assets/images/banner-image7.png" alt="thunk">
+            </div>
+        </div>
+    </div>
+</template>
+<style lang="scss" scoped src="@/components/HfHotDogdBannerHomePage/styles.scss"></style>
```

Comment:
Cần @ không z? Với lại file scss nên đặt tên theo tên component hoặc page
```suggestion
<style lang="scss" scoped src="./styles.scss"></style>
```